README.md

```diff
@@ -1,58 +1,30 @@
 
-# elrrservices     
-Elrr services which provide a mechanism to synchronize the data in the ELRR with the data in the local learning systems 
+# elrrservices
+Elrr services which provide a mechanism to synchronize the data in the ELRR with the data in the local learning systems
 
-## Getting Started
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+Setup elrrdatasync first [README](../elrrdatasync/README.md)
```

Comment:
that's not how github docs links work. You need to make the url the web url of the actual repo, it doesn't have a concept of folders and this organization just happens to be the case on your local.